test-project-setup.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+setup () {
+  # TEAM FOLDER must be in set in local environment
+  if [[ ! ${TEAM_FOLDER} ]]
```

Comment:
On the notion we want to allow developers to use set up and run the tests themselves, maybe `TEAM_FOLDER` should be optional but if absent report a message like: "`"If you are creating this project in an organization there may be a policy on project folders. To move the project after completion do X, Y, Z."